frontend/src/global_styles/layout/_toolbar.sass

```diff
@@ -204,9 +204,9 @@ $nm-color-success-background: #d8fdd1
 // Allow title container to grow
 .title-container
   flex: 1 1
-  overflow: hidden
   white-space: nowrap
   margin-bottom: 10px
+  max-width: 100%
```

Comment:
The overflow: hidden at the top was removed so the fatter "border" on the title input field isn't cut off. However, that made the ellipsis break, which is where the max-width comes in to fix the sizing behaviour.